src/compiler/scala/tools/nsc/settings/MutableSettings.scala

```diff
@@ -112,7 +112,7 @@ class MutableSettings(val errorFn: String => Unit)
   /** A list of settings which act based on prefix rather than an exact
    *  match.  This is basically -D and -J.
    */
-  lazy val prefixSettings = allSettings collect { case x: PrefixSetting => x }
+  lazy val prefixSettings = allSettings.valuesIterator.collect { case x: PrefixSetting => x }.toList
```

Comment:
Thanks again for restoring dots when you think of it. This isn't even a dotty pun. For a while it seemed cool that you didn't need dots and also you couldn't type method names on your phone because they were symbolic. I think Napoleon said an army travels on its stomach? Computer science advances on its syntax. Oh, google says maybe the saying via Carlyle is, "an army, like a serpent, goes upon its belly." It harms the pun, but syntax slithering on like a snake seems apt to me.